examples/mnist/run.py

```diff
@@ -57,43 +64,33 @@ def normalize(image, label):
   def _build_training_graph(self, training_data):
     """Build a graph for plaintext model training."""
 
-    # model parameters and initial values
-    w0 = tf.Variable(tf.random_normal([28 * 28, 512]))
-    b0 = tf.Variable(tf.zeros([512]))
-    w1 = tf.Variable(tf.random_normal([512, 10]))
-    b1 = tf.Variable(tf.zeros([10]))
-
-    # optimizer and data pipeline
-    optimizer = tf.train.AdamOptimizer(learning_rate=0.01)
-
-    # training loop
-    def loop_body(i):
-
-      # get next batch
-      x, y = training_data.get_next()
+    model = keras.Sequential()
+    model.add(keras.layers.Flatten())
+    model.add(keras.layers.Dense(512, input_shape=[28, 28, 1]))
+    model.add(keras.layers.Activation('relu'))
+    model.add(keras.layers.Dense(self.NUM_CLASSES, activation='softmax'))
```

Comment:
the softmax here should be removed (the loss function you're using expects logits and not probabilities)